.commitlintrc.js

```diff
@@ -16,5 +16,57 @@
 
 module.exports = {
   // https://www.conventionalcommits.org/en/v1.0.0-beta.2/
-  extends: ['@commitlint/config-conventional']
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'scope-case': [2, 'always', 'snake-case'],
+    'scope-empty': [1, 'never'],
+    'scope-enum': [
```

Comment:
Maybe an example comment on how scopes are used

// fix(extraction): or whatever

Reply:
That documentation is at the link above.